Assets/Scripts/Fire.cs

```diff
@@ -0,0 +1,38 @@
+﻿using UnityEngine;
+
+public class Fire : MonoBehaviour {
+    public float scaleFire;
+    public float speedDreasing;
```

Comment:
No idea what happened here but it is still wrong, it is 'speedDecreasing'

Reply:
Yes I am just stupid

Reply:
Hahaha don't worry about it, we all are